vuepress/docs/next/docs/compose/local-hub-overview.md

```diff
@@ -1,67 +1,53 @@
 
-The following examines some of the nuts and bolts of the Local Hub.
+The functional blocks and services of the Local Hub are examined below in further detail.
 
-### Component
+### Entando Component
 
-An Entando component - simply referred to as component - is an identifiable resource or block of
-code that can be used in an Entando widget, page or
-application. Examples of components are widgets, micro frontends,
-content types, labels, plugins, and static resources.
+Entando defines a component as an identifiable resource or block of code that can be used in an Entando Application. Examples of components are widgets, micro frontends, content types, labels, plugins, and static resources.
 
 ### Entando Bundle
 
-An Entando Bundle is a package containing one or more components and descriptor files that provide information about the bundle. A git-based bundle requires a `descriptor.yaml` and is published in a Git registry. A docker-based bundle requires an `entando.json` and is published to Docker. Both git-based and docker-based bundles are shared with an Entando Application using the EntandoDeBundle custom resource.
+An [Entando Bundle](../getting-started/concepts-overview.md#entando-bundle) is a package containing one or more components, the required descriptor files, and bundle metadata.
+
+A git-based bundle requires a `descriptor.yaml` and is published in a Git registry. A docker-based bundle requires an `entando.json` and is published to Docker. Both git-based and docker-based bundles are shared with an Entando Application using the EntandoDeBundle custom resource.
 
 ### EntandoDeBundle Custom Resource
 
-The EntandoDeBundle custom resource is a Kubernetes resource
-readable by the Entando Operator. It provides information
-about an Entando Bundle and makes the bundle available in Kubernetes for the
-Entando Component Manager.
+The EntandoDeBundle custom resource is a Kubernetes resource read by the Entando Operator. It provides information about an Entando Bundle and makes the bundle available to the Entando Component Manager in Kubernetes.
 
-### Entando Component Manager (ECM)
+### Entando Component Manager
 
-The Entando Component Manager is part of the
-EntandoApp and communicates with both the `entando-core` and Kubernetes cluster via the `entando-k8s-service`. The ECM reads the bundles from the cluster and exposes them with an API accessible from the App Builder. It is
-also responsible for the installation and removal of components from the `entando-core`. 
+The [Entando Component Manager](../compose/ecm-overview.md) creates the connection between the EntandoApp
+and the Entando Kubernetes integration service. It is part of the EntandoApp and communicates with both the `entando-core` and Kubernetes cluster. It is also responsible for the installation and removal of components from the `entando-core`. 
```

Comment:
both the `entando-core` and Kubernetes cluster >> both the `entando-core` and the Kubernetes cluster.

add the before Kubernetes